lib/process-services-cloud/src/lib/form/components/dropdown-cloud/dropdown-cloud.widget.ts

```diff
@@ -67,7 +68,20 @@ export class DropdownCloudWidgetComponent extends WidgetComponent implements OnI
         });
      }
 
-    getOptionValue(option: FormFieldOption, fieldValue: string): string {
+     private setOptionValues(result: any) {
```

Comment:
missing type information
```suggestion
     private setOptionValues(result: { id: string; name: string }) {
```